build-constraints.txt

```diff
@@ -1,6 +1,2 @@
 # build version constraints for use with wheelwright + multibuild
-numpy==1.15.0; python_version<='3.7' and platform_machine!='aarch64'
-numpy==1.19.2; python_version<='3.7' and platform_machine=='aarch64'
-numpy==1.17.3; python_version=='3.8' and platform_machine!='aarch64'
-numpy==1.19.2; python_version=='3.8' and platform_machine=='aarch64'
```

Comment:
Keeping the line below is a bit weird, but the constraints are probably needed again in the future, so I think it's nice to keep around this file and its history for the future?

Reply:
Sure!